contrib/gonic/files/gonic

```diff
@@ -0,0 +1,5 @@
+type = process
+command = /usr/bin/gonic -config-path /etc/gonic/config
+depends-on = network.target
```

Comment:
network.target is only supplementary; you want to also add depends-on = local.target